src/Elliptic/Systems/ScalarGaussBonnet/Equations.hpp

```diff
@@ -0,0 +1,172 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Elliptic/Systems/ScalarGaussBonnet/Tags.hpp"
+#include "Elliptic/Systems/Xcts/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+namespace sgb {
+struct Fluxes;
+struct Sources;
+}  // namespace sgb
+/// \endcond
+
+namespace sgb {
+
+/*!
+ * \brief Compute the fluxes \f$F^i=\gamma^{ij}\partial_j u(x)\f$ for the scalar
```

Comment:
FYI, you don't need the `\f`, standard `$` latex syntax works as well (I'm not requesting you change anything).

Reply:
I've removed them anyway for cleanliness.